CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
```

Comment:
nit: maybe `pledge to make` would be more correct?

Reply:
as in the original: https://www.contributor-covenant.org/version/1/4/code-of-conduct/